resources/sync_gateway_configs/sync_gateway_default_functional_tests_no_port_cc.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
Do we want to include couchbase in the filename?

Reply:
Do you mean that filename to include couchbase? It is better to have more generic file names , so that we can use for other use cases(other than couchbase usecase) if needed later